packages/data-service/src/csfle-collection-tracker.spec.ts

```diff
@@ -360,7 +381,7 @@ describe('CSFLECollectionTracker', function () {
           dataService.findOneAndUpdate(
             `${dbName}.test2`,
             {},
-            { $set: { a: 2 } },
+            { $set: { a: '2' } },
```

Comment:
csfle library was, very understandably, complaining about the fact that the schema above says this should be a `string` :)